addons/sale/models/product_template.py

```diff
@@ -239,8 +239,17 @@ def _get_combination_info(self, combination=False, product_id=False, add_qty=1,
             price = product.price if pricelist else list_price
             display_image = bool(product.image_1920)
             display_name = product.display_name
+            if no_variant_attributes_price_extra and pricelist:
```

Comment:
Is this just for the tests to go through? Having `no_variant_attributes_price_extra` shouldn't be possible, no?
If so, definitely worth a comment to explain that IMHO.

As I mentioned in my previous message: 
> But as far as I can tell, this was prevented by 054baf9 so it seems ok to ignore that case.

Reply:
It is possible to have a `no_variant_attributes_price_extra` : when you have a variant attribute with `Variants Creation Mode` set to `Never` and a `price_extra` set.